db/seeds.rb

```diff
@@ -5,3 +5,138 @@
+  url = URI.parse(end_point)
+
+  http = Net::HTTP.new(url.host, url.port)
+  request = Net::HTTP::Get.new(url)
+
+  response = http.request(request)
+  JSON.parse(response.body)
+end
+
+def load_city_airports
+# Create city_airports
+  aviation = aviationstack_data(0)
+  airports = aviation["data"]
+  pagination = aviation["pagination"]
+
+  pages = (pagination["total"]/pagination["limit"]).floor
+  limit = pagination["limit"]
+
+  puts "Total pages: #{pages}"
+
+  page = 0
+  offset = 0
+  count = 0
+
+  pages = 10
+
+  loop do
+    puts "Page: #{page}"
+    aviation = aviationstack_data(offset)
+    airports = aviation["data"]
+
+    airports.each do |airport|
+      count += 1
+
+      city = City.new
+      city.name = airport["airport_name"]
+      city.country = airport["country_name"]
+      city.latitude = airport["latitude"]
+      city.longitude = airport["longitude"]
+      city.airport_code = airport["iata_code"]
+      city.gmt = airport["gmt"]
+      city.city_iata_code = airport["city_iata_code"]
+      city.country_iso2 = airport["country_iso2"]
+      city.airport_name = airport["airport_name"]
+      city.country_name = airport["country_name"]
+      city.timezone = airport["timezone"]
+
+      city.save
+
+      puts "#{count}. #{airport["iata_code"]} in #{airport["country_name"]}" if city.id
+    end
+
+    break if page == pages
+    page += 1
+    offset = (page) * limit
+
+  end
+end
+
+def create_users
+# Create users
+  user_array = [
+    {firstname: "Pat", lastname: "Sharp", address1: "89 High Street", address2: "Teddington", postcode: "TW11 8HG"},
+    {firstname: "Bob", lastname: "Hope", address1: "18 Teddington Park Rd", address2: "Teddington", postcode: "TW11 0AQ"},
+    {firstname: "Kylie", lastname: "Manogue", address1: "70 London Rd", address2: "Kingston upon Thames", postcode: "KT2 6PY"},
+    {firstname: "Cyril", lastname: "Shepard", address1: "210 Kingston Rd", address2: "Teddington", postcode: "TW11 9JF"},
+    {firstname: "Bruce", lastname: "Willis", address1: "113 Heath Rd", address2: "Twickenham", postcode: "TW1 4AZ"},
+    {firstname: "Knight", lastname: "Rider", address1: "427 Richmond Rd", address2: "Twickenham", postcode: "TW1 2EF"},
+    {firstname: "Metal", lastname: "Mickey", address1: "11 Upper Teddington Rd", address2: "Hampton Wick", postcode: "KT1 4DL"},
+    {firstname: "Roland", lastname: "Rat", address1: "19 Upper Ham Rd", address2: "Ham", postcode: "KT2 5QX"}
+    ]
+
+  user_array.each do |user|
+    p "Creating #{user[:firstname]} #{user[:lastname]}"
+    city = City.all.sample
+    u = User.new
+    u.first_name = user[:firstname]
+    u.last_name = user[:lastname]
+    u.email = "#{user[:firstname]}.#{user[:lastname]}@upintheair.com"
+    u.password = "topsecret"
+    u.address_1 = user[:address1]
+    u.address_2 = user[:address2]
+    u.postcode = user[:postcode]
+    u.city_id = city.id
```

Comment:
I think you can do 
```ruby
User.new(user)
```
since `user` is a hash. Will need to add the password separately still.

Reply:
Okay, makes sense. Will do when local as part of refactoring.

Reply:
Will updated locally. I can't do a push at the moment, as I need to pull due to merge conflict. Can you approve the PR, so I can pull down and resolve it.